### PR TITLE
Organization Detail Page added

### PR DESCRIPTION
**Pull request by Kritika-30**
Hello @devesh-verma 

Added another page which depicts the detail of Organization. This page is accessible through 'see more' button in Organization Tab.

Along with this, a data folder has been added which contains all the files having data of user and organization.
Let us know your feedback on this.

Organization Detail

### CODE REVIEW

**Comment by jaskiratsingh2000** [comment]
Looks good to me.

File: client/src/App.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
     return (
       <BrowserRouter>
+	  <Navbar />
         <div className="App">
           <Switch>
-             <Route exact path='/'component={Navbar} /> 
-             {/* <Route path='/organizations' component={Organization} />  */}
+             <Route exact path='/Dashboard' component={DashboardContainer} /> 
+             <Route exact path='/organization' component={OrganizationContainer} />
+			 <Route path='/OrganizationDetail' component = {DetailContainer} />
```
Comment (Rupeshiya) [inline client/src/App.js:20]: use exact path

File: client/src/Components/Layout/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+  const history = useHistory();
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  const handleClick = () => {
+    history.push('./Dashboard');
```
Comment (Rupeshiya) [inline client/src/Components/Layout/Navbar.jsx:32]: why ./dashboard , I think it should be /dashboard?

File: client/src/Components/Layout/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    
+  };
+
+  const handle = () => {
+    history.push('./Organization');
```
Comment (Rupeshiya) [inline client/src/Components/Layout/Navbar.jsx:37]: same here

File: client/src/Components/Organization/OrganizationCard.jsx [hunk 4]
```diff
@@ -35,8 +36,15 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
+
 export default function ImgMediaCard() {
   const classes = useStyles();
+  const history = useHistory();
+
+  function handleChange(){
+    history.push('./OrganizationDetail');
```
Comment (Rupeshiya) [inline client/src/Components/Organization/OrganizationCard.jsx:45]: same here

**Comment by ayushnagar123** [comment]
image
image
image

* The page doesn't look appropriately alligned as per the design please center the content of the page and look trough all the screen sizes >=768 px for desktop view in all screens.
* View more on each card can have a bit more margin bottom.
* On hover not whole of the card shows effect which looks odd please check that out
 Please also look through the changes proposed by Rupeshia in the code.

File: client/src/Components/Organization/OrganizationCard.jsx (hunk 4 above)
Comment (Kritika-30) [reply client/src/Components/Organization/OrganizationCard.jsx:45]: Thanks Rupesh for the suggestions. I have made the required changes.

**Comment by Kritika-30** [comment]
> image
> image
> image
> 
> * The page doesn't look appropriately alligned as per the design please center the content of the page and look trough all the screen sizes >=768 px for desktop view in all screens.
> * View more on each card can have a bit more margin bottom.
> * On hover not whole of the card shows effect which looks odd please check that out
>   Please also look through the changes proposed by Rupeshia in the code.

Thanks @ayushnagar123 for suggesting the changes. I am working on it. 

I wanted to know what we should prefer for responsive design in react projects. Should we use Materail-css or something else?

**Comment by ayushnagar123** [comment]
Use material-css its good and all our projects are designed as per material-css.

**Pull request merged**